pkgs/by-name/gi/git-autoshare/package.nix

```
@@ -0,0 +1,28 @@
+{ fetchFromGitHub
+, lib
+, python3Packages
+}:
+python3Packages.buildPythonApplication rec {
+  pname = "git-autoshare";
+  version = "1.0.0b6";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "acsone";
+    repo = "git-autoshare";
+    rev = version;
+    sha256 = "sha256-F8wcAayIR6MH8e0cQSwFJn/AVSLG3tVil80APjcFG/0=";
+  };
+
+  doCheck = false;
+
+  build-system = with python3Packages; [ setuptools-scm ];
+  dependencies = with python3Packages; [ appdirs click pyyaml ];
+
```

```suggestion

  doCheck = false;

```
a comment explaining why these are disabled would also be nice -- i'm guessing it's because they attempt to connect to github?